runtimes/spiritnet/src/filter.rs

```diff
@@ -0,0 +1,68 @@
+use frame_support::traits::Contains;
+
+use super::Call;
+
+pub struct GovCalls;
```

Comment:
C: Move to `common`.

Reply:
Would love to but i don't think this is possible since it's dependent on the runtimes Call. :(